module/dev_functions.py

```diff
@@ -1,77 +1,73 @@
-import yaml
-import telegram
-import os
 import logging
+from datetime import datetime
 
 from telegram import Update
 from telegram.ext import CallbackContext
 
-from datetime import datetime
-
 from module.config import load_configurations
 
 config_map = load_configurations()
 
+
 def logging_message(update: Update, context: CallbackContext):
-	try:
-		message_id = update.message.message_id #ID MESSAGGI\O
-		user = update.message.from_user # Restituisce un oggetto Telegram.User
-		chat = update.message.chat # Restituisce un oggetto Telegram.Chat
-		text = update.message.text #Restituisce il testo del messaggio
-		date = update.message.date #Restituisce la data dell'invio del messaggio
-		message = "\n___ID MESSAGE: " 	+ str(message_id) + "____\n" + \
-					"___INFO USER___\n" + \
-					"user_id:" 			+ str(user.id) + "\n" + \
-					"user_name:" 			+ str(user.username) + "\n" + \
-					"user_firstlastname:" + str(user.first_name) + " " + str(user.last_name) + "\n" + \
-					"___INFO CHAT___\n" + \
-					"chat_id:" 			+ str(chat.id) + "\n" + \
-					"chat_type:" 			+ str(chat.type)+"\n" + "chat_title:" + str(chat.title) + "\n" + \
-					"___TESTO___\n" + \
-					"text:" 				+ str(text) + \
-					"date:" 				+ str(date) + \
-					"\n_____________\n"
-
-		log_tmp = open("logs/logs.txt","a+")
-		log_tmp.write("\n"+message)
-	except:
-		pass
+    try:
+        message_id = update.message.message_id  #ID MESSAGGI\O
+        user = update.message.from_user  # Restituisce un oggetto Telegram.User
+        chat = update.message.chat  # Restituisce un oggetto Telegram.Chat
+        text = update.message.text  #Restituisce il testo del messaggio
+        date = update.message.date  #Restituisce la data dell'invio del messaggio
+        message = "\n___ID MESSAGE: "  + str(message_id) + "____\n" + \
+           "___INFO USER___\n" + \
+           "user_id:"    + str(user.id) + "\n" + \
+           "user_name:"    + str(user.username) + "\n" + \
+           "user_firstlastname:" + str(user.first_name) + " " + str(user.last_name) + "\n" + \
+           "___INFO CHAT___\n" + \
+           "chat_id:"    + str(chat.id) + "\n" + \
+           "chat_type:"    + str(chat.type)+"\n" + "chat_title:" + str(chat.title) + "\n" + \
+           "___TESTO___\n" + \
+           "text:"     + str(text) + \
+           "date:"     + str(date) + \
+           "\n_____________\n"
+
+        log_tmp = open("logs/logs.txt", "a+")
+        log_tmp.write("\n" + message)
+    except Exception:
+        pass
```

Comment:
I would suggest to print the exception

Reply:
It is saved in the _logs/logs.txt_ file

Reply:
no if it fails

Reply:
I mean, the whole method is very debug-oriented: it just logs all messages the bot receives in the file.
It doesn't do anything else, and is used only one time, in a MessageHandler

Reply:
ok, I am too much pedantic